Source/Scene/MetadataSemantic.js

```diff
@@ -57,6 +57,86 @@ var MetadataSemantic = {
    * @private
    */
   ID: "ID",
+  /**
+   * A bounding box for a tile, stored as an array of 12 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are the same format as for <code>boundingVolume.box</code> in 3D Tiles 1.0. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  TILE_BOUNDING_BOX: "TILE_BOUNDING_BOX",
+  /**
+   * A bounding region for a tile, stored as an array of 6 <code>FLOAT32</code> or <code>FLOAT64</code>. The components are <code>[west, south, east, north, minimumHeight, maximumHeight]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```

Comment:
```suggestion
   * A bounding region for a tile, stored as an array of 6 <code>FLOAT64</code> components. The components are <code>[west, south, east, north, minimumHeight, maximumHeight]</code>. This semantic is used to provide a tighter bounding volume than the one implicitly calculated in <code>3DTILES_implicit_tiling</code>
```

Region components must be float64 for precision reasons:
https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata/Semantics#tile_bounding_region